packages/core/src/App/Containers/Redirect/redirect.jsx

```diff
@@ -117,6 +120,7 @@ const Redirect = ({
             break;
         }
         case 'payment_withdraw': {
+            setLoginId(loginid_param);
```

Comment:
Just a question 🤔 Wouldn't it be better to always update the loginid if it exists in the URL instead of just doing it for `payment_withdraw` case? 🤔 That would cover more cases which hasn't been raised yet 🤔